aptos-move/e2e-testsuite/Cargo.toml

```diff
@@ -19,7 +19,7 @@ aptos-crypto = { workspace = true }
 aptos-framework = { workspace = true }
 aptos-gas-algebra = { workspace = true }
 aptos-gas-meter = { workspace = true }
-aptos-gas-schedule = { workspace = true }
+aptos-gas-schedule = { workspace = true, features = ["testing"] }
```

Comment:
fixes this issue https://github.com/aptos-labs/aptos-core/pull/9501#discussion_r1284876505